src/WebJobs.Script/Workers/ProcessManagement/WorkerProcess.cs

```diff
@@ -200,7 +201,12 @@ public void Dispose()
                     if (!_process.HasExited)
                     {
                         _process.Kill();
-                        _process.WaitForExit();
+                        _process.WaitForExit(processExitTimeoutInMilliseconds);
```

Comment:
> _process.WaitForExit(processExitTimeoutInMilliseconds); [](start = 23, length = 56)

Unfortunately, this and other _process methods are non-virtual so was not immediately able to add tests for these. Not unless we do some kind of refactor here where we have some kind of an abstraction and have WorkerProcess take dependency on that abstraction of process instead of directly on the Process.